fetch/api/abort/general.js

```diff
@@ -0,0 +1,335 @@
+if (self.importScripts) {
+  // Load scripts if being run from a worker
+  importScripts(
+    '/resources/testharness.js',
+    '/common/utils.js'
+  );
+}
+
+// This is used to close connections that weren't correctly closed during the tests,
+// otherwise you can end up running out of HTTP connections.
+let requestKeys = [];
+
+function abortRequests() {
+  const keys = requestKeys;
+  requestKeys = [];
+  return Promise.all(
+    keys.map(key => fetch(`../resources/stash-put.py?key=${key}&value=close`))
+  );
+}
```

Comment:
Move requestKeys and abortRequests done to where they are actually used? I guess I can kinda see they are global, but to me at least I kept wondering when they'd show up and when they did I had forgotten the setup.